web/libs/datamanager/src/components/Common/MediaPlayer/MediaSeeker.scss

```
@@ -0,0 +1,50 @@
+.audio-seeker {
+  height: 5px;
+  width: 100%;
+  position: relative;
+  border-radius: 0 0 4px 4px;
+  background-color: rgba(#000, 0.05);
```

Flagging as opportunity for css var replacement. Further review of usage needed.
```suggestion
  background-color: rgba(var(--black), 0.05);
```